src/pages/Admin/Charity/Withdraws/Withdrawer/WithdrawForm/Amounts.tsx

```diff
@@ -29,40 +30,40 @@ export default function Amounts() {
 
         return (
           <div
-            className="flex relative mb-6 border-b border-prim pr-2 pb-1 pt-6 items-center"
+            className="relative grid grid-cols-[auto_1fr_auto] items-center gap-2 @md:gap-4 w-full p-4 border border-prim rounded"
             key={field.id}
           >
-            <button
-              onClick={() => {
-                setValue(fieldName, field.balance, {
-                  shouldValidate: true,
-                  shouldDirty: true,
-                });
-              }}
-              type="button"
-              className="absolute top-1 right-2 text-blue dark:text-blue-l3 uppercase text-xs"
-            >
-              bal: {humanize(+field.balance, 4)}
-            </button>
-            <img
-              src={tokens[field.tokenId].icon}
-              alt=""
-              className="h-5 w-5 object-contain mr-1"
-            />
             <label
               htmlFor={field.id}
-              className="uppercase font-heading text-sm"
+              className="flex items-center gap-3 h-full uppercase"
             >
+              <Logo
```

Comment:
@misicnenad, I think this logo is causing this flicker
https://www.loom.com/share/cb96396d0adc43f1b697117f8c9a1807

Reply:
Taking a look

Reply:
This was an issue with the Logo loading logic - even when the source img is already loaded, because `isLoading` is set to `true` by default, it causes the `ContentLoader` to be displayed and the component is forced to wait until next rendering cycle to render the image itself. This and also the `hidden` class should have been applied to `WithLink` to avoid the left-to-right-and-back flicker.

Moved the `isLoading ? "hidden" : ""` condition up to `WithLink` and added a check to `HTMLImageElement.complete?: boolean` prop to see if the image was already loaded and if so, `ContentLoader` will never appear, thus avoiding the flicker https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/1876/commits/eae289aa18542c947c0681bf3ce52436e6f5175a.